### PR TITLE
Update cluster UserGuide and Config-Manual

### PR DESCRIPTION
**Pull request by qiaojialin**
Update cluster UserGuide: Cluster-Concept , Cluster-Setup

Config-Manual -> ConfigNode-Config-Manual, DataNode-Config-Manual

### CODE REVIEW

File: docs/UserGuide/Reference/DataNode-Config-Manual.md [hunk 1]
```diff
@@ -19,39 +19,32 @@
...
 In the parameters described below, these parameters whose way of `Effective` is `trigger` support hot modification.
 
-Trigger way: The client sends the command `load configuration` to the IoTDB server. See Chapter 4 for the usage of the client.
+Trigger way: The client sends the command(sql) `load configuration` to the IoTDB server.
 
-## IoTDB Environment Configuration File
+## Environment Configuration File（datanode-env.sh/bat）
 
-The environment configuration file is mainly used to configure the Java environment related parameters when IoTDB Server is running, such as JVM related configuration. This part of the configuration is passed to the JVM when the IoTDB Server starts. Users can view the contents of the environment configuration file by viewing the `datanode-env.sh` (or `datanode-env.bat`) file.
+The environment configuration file is mainly used to configure the Java environment related parameters when ConfigNode is running, such as JVM related configuration. This part of the configuration is passed to the JVM when the DataNode starts.
```
Comment (HTHou) [inline docs/UserGuide/Reference/DataNode-Config-Manual.md:39]:
```suggestion
The environment configuration file is mainly used to configure the Java environment related parameters when DataNode is running, such as JVM related configuration. This part of the configuration is passed to the JVM when the DataNode starts.
```

File: docs/UserGuide/Cluster/Cluster-Setup.md [hunk 2]
```diff
@@ -19,376 +19,103 @@
...
+## Environments
 
-1. Java >= 1.8 (Please make sure the environment path has been set)
+1. JDK>=1.8
 
-2. Set the max open files num as 65535 to avoid "too many open files" problem.
+2. Max open file 65535
 
-   
+3. Disable the swap memory
```
Comment (HTHou) [inline docs/UserGuide/Cluster/Cluster-Setup.md:32]: Why should we disable the swap memory?

File: docs/UserGuide/Reference/DataNode-Config-Manual.md [hunk 3]
```diff
@@ -1131,14 +1247,14 @@ The permission definitions are in ${IOTDB\_CONF}/conf/jmx.access.
...
 
-To enable GC log, just add a parameter "printgc" when you start the server.
+To enable GC log, just add a parameter "printgc" when you start the DataNode.
 
 ```bash
-nohup sbin/start-server.sh printgc >/dev/null 2>&1 &
+nohup sbin/start-datanode.sh printgc >/dev/null 2>&1 &
 ```
 Or
 ```bash
```
Comment (HTHou) [inline docs/UserGuide/Reference/DataNode-Config-Manual.md:1256]:
```suggestion
```cmd
```

File: docs/UserGuide/Cluster/Cluster-Setup.md (hunk 2 above)
Comment (qiaojialin) [reply docs/UserGuide/Cluster/Cluster-Setup.md:32]: To avoid system fluctuation even if the memory of IoTDB looks enough.

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**